tests/common

```diff
@@ -0,0 +1,46 @@
+#!/bin/ksh
+set -e 
+failed=0
+
+workdir="$PWD"
+tmpdir=$(mktemp -d -p $workdir)
+
+port=13245
+www=$(mktemp -d -p $tmpdir)
```

Comment:
Is there any reason for random name here? Not saying this is an error, just curious.